packages/expo-av/ios/EXAV/EXAV+AudioSampleCallback.h

```diff
@@ -0,0 +1,19 @@
+//
+//  EXAV+AudioSampleCallback.h
+//  EXAV
+//
+//  Created by Marc Rousavy on 09.07.21.
+//
```

Comment:
same as above, use expo copyright, remove file name and pod to patch other files